DigitalLearningSolutions.Web/Controllers/LearningMenuController/LearningMenuController.cs

```diff
@@ -264,5 +267,48 @@ public IActionResult TutorialVideo(int customisationId, int sectionId, int tutor
             );
             return View("Tutorial/TutorialVideo", model);
         }
+
+        [Route("/LearningMenu/{customisationId:int}/CompletionSummary")]
+        public IActionResult CompletionSummary(int customisationId)
+        {
+            var candidateId = User.GetCandidateId();
+            var centreId = User.GetCentreId();
+
+            var courseCompletion = courseCompletionService.GetCourseCompletion(candidateId, customisationId);
+
+            if (courseCompletion == null || centreId == null)
+            {
+                logger.LogError(
+                    "Redirecting to 404 as customisation id was not found. " +
+                    $"Candidate id: {candidateId}, customisation id: {customisationId}, " +

```

Comment:
We should probably log the centre id here as well (and in the other similar logs), so that it's clear from the logs what's happened if the centre id is null 🙂

Reply:
Yes I agree! Done in adf3f44 🙂